demo/mfRoutes.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  'home@http://localhost:3000/_next/static/chunks/remoteEntry.js': [
```

Comment:
is this auto-generated?

Reply:
No, it isn't. But it can be. I just put it in this file for easier editing. 

It's just a Lazy map of remotes that are passed in runtime. So it might be changed even for every specific request –some users receive one map, others eg. admins extended another.